debugger.md

```diff
@@ -39,21 +39,21 @@ The first time that C# code is opened in VS Code, the extension will download th
 The following steps have to executed for every project. 
 
 ##### 1: Get a project
-You can start from scratch by creating an empty project with `dotnet new`:
+You can start from scratch by creating an empty project with `dotnet new`. Begin by opening the terminal in Visual Studio Code (`View->Integrated Terminal`) or use Command Prompt and type these commands:
 
     cd ~
```

Comment:
Good point. I'll make one more revision and remove the `or use Command Prompt` part.